### PR TITLE
POST requests: how to use a Postman collection

### PR DESCRIPTION
**Pull request by Paul-Cornell**
This enables Postman users to more quickly and easily make POST requests to Unstructured API services.

See https://unstructured-53-postman-2024-09-12.mintlify.app/api-reference/api-services/post-requests#postman-collection

### CODE REVIEW

File: api-reference/api-services/post-requests.mdx [hunk 1]
```diff
@@ -46,3 +46,49 @@ The API parameters are the same across all methods of accessing the Unstructured
...
+
+import UseTheCorrectAPIKey from '/snippets/general-shared-text/api-key-differences.mdx';
+
+Unstructured offers a [Postman collection](https://learning.postman.com/docs/collections/collections-overview/) that you can import into Postman to make POST requests through a graphical user interface.
+
+1. [Install Postman](https://learning.postman.com/docs/getting-started/installation/installation-and-updates/).
+2. [Sign in to Postman](https://learning.postman.com/docs/getting-started/installation/postman-account/#signing-in-to-postman).
+3. In your workspace, click **Import**.
+
+   ![](/img/api/post/import.png)
```
Comment (MKhalusova) [inline api-reference/api-services/post-requests.mdx:60]: Let's add alt text for all the images

Comment (Paul-Cornell) [reply api-reference/api-services/post-requests.mdx:60]: Oops. I meant to add the alt text (hence the `[]` characters) but forgot to do it before pushing. Thanks for the catch!

File: api-reference/api-services/post-requests.mdx [hunk 2]
```diff
@@ -46,3 +46,49 @@ The API parameters are the same across all methods of accessing the Unstructured
...
+
+6. Expand **Unstructured POST**.
+7. Click one of the following:
+
+   - To call the Unstructured Serverless API, click **(Serverless API) Basic Request**.
+   - To call the Free Unstructured API, click **(Free API) Basic Request**.
+
+8. On the **Headers** tab, next to `unstructured-api-key`, enter your Unstructured API key in the **Value** column.
+
+   ![](/img/api/post/headers.png)
```
Comment (Paul-Cornell) [inline api-reference/api-services/post-requests.mdx:80]:
```suggestion
   ![Headers tab in the request area](/img/api/post/headers.png) 
```

File: api-reference/api-services/post-requests.mdx [hunk 3]
```diff
@@ -46,3 +46,49 @@ The API parameters are the same across all methods of accessing the Unstructured
...
+
+8. On the **Headers** tab, next to `unstructured-api-key`, enter your Unstructured API key in the **Value** column.
+
+   ![](/img/api/post/headers.png) 
+
+   <UseTheCorrectAPIKey />
+
+9. On the **Body** tab, next to `files`, click the **Select files** box in the **Value** column.
+
+   ![](/img/api/post/body.png)
```
Comment (Paul-Cornell) [inline api-reference/api-services/post-requests.mdx:86]:
```suggestion
   ![Body tab in the request area](/img/api/post/body.png) 
```

File: api-reference/api-services/post-requests.mdx [hunk 4]
```diff
@@ -46,3 +46,49 @@ The API parameters are the same across all methods of accessing the Unstructured
...
+
+   ![](/img/api/post/body.png) 
+
+11. Click **New file from local machine**.
+12. Browse to and select the file that you want Unstructured to process. 
+13. Click **Send**.
+
+To download the processed data to your local machine, in the response area, click the ellipses, and then click **Save response to file**.
+
+![](/img/api/post/save.png)
```
Comment (Paul-Cornell) [inline api-reference/api-services/post-requests.mdx:94]:
```suggestion
![Save response to file in the response area](/img/api/post/save.png)
```

**Pull request merged**